### ISSUE TITLE
Searching feature?

### ISSUE THREAD

**Issue by garak** [israel-munoz/multiselect-dropdown#6]
I couldn't find a configuration for searching among select options, so I guess it's a missing feature.
What about implementing it?

TIA

**Comment by israel-munoz** [comment]
Yes. Missing feature. But sounds cool. I’m short of time lately but will work in a new version and implement that if I can. Thank you so much for the suggestion.
